### PR TITLE
Add docs on how to upgrade the metrics components between versions.

### PR DESCRIPTION
**Pull request by mwringe**

### CODE REVIEW

**Comment by mwringe** [comment]
Bump, any news on this?

**Comment by tnguyen-rh** [comment]
@mwringe This is for 3.2 only, right?

**Comment by mwringe** [comment]
@tnguyen-rh yes, its for when people are moving between 3.1 and 3.2 (since that is the only main version update they have so far). But it will also be required for going between 3.2 and 3.3, etc

**Comment by tnguyen-rh** [comment]
@mwringe Thanks.  For future reference, feel free to add the label `branch/enterprise-3.2` directly, and tag me or @openshift/team-documentation to call (more) timely attention to the PR.

**Comment by adellape** [comment]
@tnguyen-rh Added a milestone.

**Comment by mwringe** [comment]
Any idea when this will appear in the 3.2 docs? I can see it in the origin docs but not the 3.2 ones.

**Comment by bfallonf** [comment]
@mwringe Once a PR is merged it's in Origin fairly quickly, but isn't released until the followup label (to double-check if it hits all the CCS standards) is removed and after our weekly async release.

Does this need to be in the Enterprise docs quickly?

**Comment by tnguyen-rh** [comment]
@mwringe To expand on what @bfallonf said, the `to_followup` label will be removed once #2234 is merged.

File: install_config/upgrading/manual_upgrades.adoc [hunk 1]
```diff
@@ -908,6 +908,58 @@ in previous versions. There should be minimal performance impact to
...
+
+----
+$ oc new-app -f metrics-deployer.yaml \
+    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com
+----
+
+You will now need to run:
+----
+$ oc new-app -f metrics-deployer.yaml \
+    -p HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,MODE=redeploy
```
Comment (tnguyen-rh) [inline install_config/upgrading/manual_upgrades.adoc]: @mwringe 
Hmm, there is a discrepency between this example (which uses `redeploy`) and the introductory text (which uses `refresh`).  Which is correct?  (I will make the change in #2234.)

Comment (mwringe) [inline install_config/upgrading/manual_upgrades.adoc]: Oops, it should be 'refresh' not 'redeploy' here. Thanks for the catch

**Comment by tnguyen-rh** [comment]
PR #2234 has rev history and has merged.  Removing `to_followup` label.

**Comment by mwringe** [comment]
@adellape I still don't see this in the OSE 3.2 docs, this should have been a high priority fix as its essential for anyone updating to OSE 3.2 who had metrics deployed before.

**Comment by adellape** [comment]
@mwringe It's in the Staging milestone currently, meaning we're currently working on cherry-picking and publishing to OSE docs. I'll bump you when it's live.

**Comment by adellape** [comment]
@mwringe This is live now for OSE 3.2:

https://docs.openshift.com/enterprise/3.2/install_config/upgrading/manual_upgrades.html#manual-upgrading-cluster-metrics

**Comment by mwringe** [comment]
@adellape awesome, thanks so much!

**Pull request merged**